demo/js/components/victory-voronoi-container-demo.js

```diff
@@ -180,7 +180,7 @@ class App extends React.Component {
                   <VictoryTooltip
                     y={150}
                     flyoutStyle={{ fill: "white" }}
-                    style={[{ fill: "green" }, { fill: "magenta" }]}
```

Comment:
arrays of objects should be supported for `VictoryLabel` and `VictoryTooltip`

Reply:
going to fix the `style` prop in `VictoryTooltipProps` interface to take into account the arrays of objects